src/main/java/org/kainos/ea/controller/JobRoleController.java

```diff
@@ -35,4 +41,19 @@ public Response getAllJobRoles() {
             return Response.serverError().build();
         }
     }
+
+    @GET
+    @Path("/job_roles/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getJobSpecificationId(@PathParam("id") int id) {
+
+        try {
+            return Response.ok(jobRoleService.getSpecificationById(id)).build();
+        } catch (FailedToGetValidJobId e) {
+            System.err.println(e.getMessage());
+
+            return Response.status(HttpStatus.INTERNAL_SERVER_ERROR_500).build();
+        }
```

Comment:
there should be another exception scenario for the ID not existing in the DB which should return a 404